config/config.go

```diff
@@ -71,21 +71,38 @@ const (
 var configuration Config
 var rwMutex sync.RWMutex
 
+// Metrics provides metrics configuration for the Kiali server.
+type Metrics struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+	Port    int  `yaml:"port,omitempty"`
+}
+
+// Tracing provides tracing configuration for the Kiali server.
+type Tracing struct {
+	Enabled      bool   `yaml:"enabled,omitempty"`
+	CollectorURL string `yaml:"collector_url,omitempty"` // Endpoint for Kiali server traces
```

Comment:
trivial - I think we are trying to alphabetize the fields inside all of these config types 
```suggestion
	CollectorURL string `yaml:"collector_url,omitempty"` // Endpoint for Kiali server traces
	Enabled      bool   `yaml:"enabled,omitempty"`
```

Reply:
Good eye... I always miss one of these